src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -38,6 +39,7 @@ public class SearchIndexFromCatalogCollector : CommitCollector
             bool commitEachBatch,
             TimeSpan? commitTimeout,
             string baseAddress,
+            string galleryBaseAddress,
             ITelemetryService telemetryService,
```

Comment:
This should probably by a `Uri` object instead of a `string`.

Reply:
Yes, Uri is better than string.